buildpack_store_test.go

```diff
@@ -29,16 +29,15 @@ func testBuildpackStore(t *testing.T, when spec.G, it spec.S) {
 		fakeCacheManager = &fakes.CacheManager{}
 
 		buildpackStore = occam.NewBuildpackStore()
-
-		buildpackStore = buildpackStore.WithLocalFetcher(fakeLocalFetcher).
```

Comment:
I don't think we needed to remove this - we could have kept it here, and later on we could have done `buildpackStore = buildpackStore.WithTarget()`. It would only override it for the specific tests that require it.

However, i think either option is fine, so no need to revert your changes.